contrib/ipp/CMakeLists.txt

```diff
@@ -46,3 +48,5 @@ list( APPEND sources ${srcs})
 enable_language(C ASM)
 set( CMAKE_ASM_SOURCE_FILE_EXTENSIONS s )
 make_library( ipp none static )
+
+set( CMAKE_ASM_FLAGS "${CMAKE_ASM_FLAGS} -x assembler-with-cpp" )
```

Comment:
can you, please, move this line above and place right before make_library? I prefer to see settings before target definition.

But again, why we actually need `-x assembler-with-cpp`: I don't see we used this flag before?

Reply:
I could not remove one flag -DNDEBUG. This flag tells armclang that the source assembly file requires preprocessing. With this flag it stops throwing a warning

Reply:
Updated. `-DNDEBUG` is added by CMake so it can be disabled by `set( CMAKE_ASM_EXTENSIONS OFF )`